### PR TITLE
Move policy.subject.format to protocol.settings.nameFormat

### PR DESCRIPTION
**Pull request by danielzeller-okta**
Docs PR for OKTA-77359

@tthompson-okta 
@karlmcguinness-okta 
@joelfranusic-okta 
@mattloughran-okta

### CODE REVIEW

File: _source/docs/api/resources/idps.md [hunk 1]
```diff
@@ -168,6 +165,7 @@ Protocol settings for the [SAML 2.0 Authentication Request Protocol](http://docs
 | type        | SAML 2.0 protocol                                          | `SAML2`                                                  | FALSE    | TRUE     |           |           |            |
 | endpoints   | SAML 2.0 HTTP binding settings for IdP and SP (Okta)       | [SAML 2.0 Endpoints Object](#saml-20-endpoints-object)   | FALSE    | FALSE    |           |           |            |
 | algorithms  | Settings for signing and verifying SAML messages           | [SAML 2.0 Algorithms Object](#saml-20-algorithms-object) | FALSE    | FALSE    |           |           |            |
+| settings    | Dictionary for various SAML 2.0 settings                   | [SAML 2.0 Settings Object](#saml-20-settings-object)     | TRUE     | FALSE    |           |           |            |
```
Comment (karlmcguinness-okta) [inline _source/docs/api/resources/idps.md]: Advanced settings for the SAML 2.0 protocol

File: _source/docs/api/resources/idps.md [hunk 2]
```diff
@@ -412,6 +413,24 @@ XML digital signature algorithm settings for verifying `<SAMLResponse>` messages
 | scope       | Specifies whether to verify a `<SAMLResponse>` message or `<Assertion>` element XML digital signature                  | `RESPONSE`, `ASSERTION`, `ANY` | FALSE    | FALSE    |           |           |            |
 |-------------+------------------------------------------------------------------------------------------------------------------------+--------------------------------+----------+----------+-----------+-----------+------------|
 
+##### SAML 2.0 Settings Object
+
+|-------------+-----------------------------------+-------------+-----------------------------------------------------------------------------|
+| Property    | Description                       | DataType    | Nullable | Readonly | Default                                               |
+| ----------- + --------------------------------- + ----------- + -------- + -------- + ----------------------------------------------------- |
+| nameFormat  | The name identifier format to use | String      | TRUE     | FALSE    | urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified |
```
Comment (karlmcguinness-okta) [inline _source/docs/api/resources/idps.md]: This should be of enum of supported values.  There is a pattern for DataType for the enum.  We should not allow value that are not defined by enum

**Comment by karlmcguinness-okta** [comment]
Looks good, can you rebase so I can merge?

**Comment by danielzeller-okta** [comment]
Done

**Pull request merged**